NewForce-Cohort-7/tabloid-cli-emerald-pinchers
# jd-blog

# Description

Add CRUD Methods to Blog Manager (Tickets 11, 12, 13, 14)
Add Blog Details (Ticket 25)
Add methods to add, view, and remove Tags from blogs (Tickets 30, 31 and 32)
Add view Blog’s Posts (GetByBlog method in PostRepository.cs) (Ticket 37)

## Type of change

Please delete options that are not relevant.

- [ x ] New feature (non-breaking change which adds functionality)

# Testing Instructions

Should be able to add, delete, update, and view Blogs
Should be able to view blog details
Should be able to add, remove, and view Blog's tags
Should be able to find post's containing blog

# Checklist:

- [ x ] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ x ] I have commented my code, particularly in hard-to-understand areas
- [ x ] My changes generate no new warnings or errors
- [ x ] I have added test instructions that prove my fix is effective or that my feature works
